### DISCUSSION TITLE
ch1-3 강의 준비 ssh 우분투 파워쉘

### DISCUSSION THREAD

**Discussion by jinbbb** [tedilabs/fastcampus-devops#68] [Q&A] [answered]
ssh 우분투 통신 부분에서 우분투가 ifconfig도 안되고 ping도 안되고 네트워크 연결도 찾아서 해보는데 안되네요 방법이 없을까요?

**Comment by posquit0** [comment, marked as answer]
안녕하세요~
현재 상황에 대해 스크린샷이나 로그 정보와 함께 같이 공유 주시면 도움 드릴 수 있을 것 같습니다. :)

**Reply by jinbbb** [reply]
- [ ] 
![캡처](https://user-images.githubusercontent.com/95834184/170493245-644732c6-a146-42bb-8ebc-7020ba5adceb.PNG)
이런 상황인데 제가 설치를 잘 못한 건지 뭐가 뭔지 모르겠네요ㅠㅠ

**Reply by jinbbb** [reply]
![image](https://user-images.githubusercontent.com/95834184/170498048-5ff28d26-97ac-41eb-bf86-0e7dc07ff56b.png)
udate도 안되니 방법이 없습니다ㅠㅠ 네트워크 보면 vEthernet(WSL) Hyper-V virtual Ethernet Adapter도 있고 강의대로 다 진행했는데 네트워크 연결만 안되네요..

**Reply by jinbbb** [reply]
혹시 오늘 답변이 어렵나요?ㅠㅠ

**Reply by posquit0** [reply]
@jinbbb 안녕하세요~ 윈도우의 WSL 리눅스 환경에서 인터넷이 잘 동작하고 있지 않은 것 같습니다. 관련하여 검색을 해보시면 비슷한 사례들을 찾을 수 있는 것 같습니다. ㅎㅎ

- https://devkyu.tistory.com/958
- https://github.com/microsoft/WSL/issues/4435

**Reply by jinbbb** [reply]
다 초기화하고 Resolv.conf에 servername 8.8.8.8. 설정과 wsl.conf에 generateResolvConf = false 다 해봤는데 안되네요ㅜ 혹시 로컬에서도 인터넷 설정을 해야하는 것이 있나요?ㅠㅠ

**Reply by posquit0** [reply]
안녕하세요,
실습 환경의 경우 개인마다 장비와 운영체제 버전, 설정 등이 모두 다르기 때문에 모든 경우에 대해서 지원을 해드리기가 어렵습니다.. ㅠㅠ
강의에서도 전달드린 내용처럼 도커 컨테이너 기술은 리눅스 운영체제 기술이라서, Windows WSL을 사용하더라도 이는 일반적인 리눅스 환경이 아니라 추후 실습을 진행하시면서도 여러 이슈들을 또 겪을 수 있습니다.

가능하다면 다른 실습 환경을 구성하는 것으로 방향을 전환해보시는 것을 추천드립니다.

- VirtualBox, VM Ware 등을 이용하여 가상머신 상에 Ubuntu 설치
- AWS 등 퍼블릭 클라우드 이용하여 Ubuntu 서버 생성
- 남는 장비를 포맷하여 Ubuntu 운영체제 설치
